test-output flag docs (#5076)
operator-framework/operator-sdk: website/content/en/docs/testing-operators/scorecard/custom-tests.md

This is useful when debugging or writing new tests so that you can view
the test logs or the pod manifests.

### Storing scorecard test output

> The `--test-output` flag can be used when executing the `operator-sdk scorecard` command to store the output of the scorecard tests in a specific directory. Any persistent volume data will be stored in the specified local directory upon completion of the scorecard tests.

Reviewer: ```suggestion
The `--test-output` flag can be used when executing the `operator-sdk scorecard` command with a config specifying [output persistence][storage] to store the output of the scorecard tests in a specific directory. Any persistent volume data will be stored in the specified local directory upon completion of the scorecard tests.

[storage]:https://pkg.go.dev/github.com/operator-framework/api@v0.10.4/pkg/apis/scorecard/v1alpha3#Storage
```

Reviewer: Move that `[storage]` link to the bottom of this doc to where the others are.

```console
$ operator-sdk scorecard ./bundle --test-output=/mytestoutput
```

> **Note**: By default, the gathered test output will be stored in the local directory from where scorecard was executed.

Reviewer: ```suggestion
**Note**: By default, the gathered test output will be stored in `$(pwd)/test-output`.
```
